src/IssueCreatorShared/Logging/FileLogger.cs

```diff
@@ -22,10 +22,11 @@ public FileLogger(string logPath)
 
         public LogScope CreateScope(string message)
         {
+            
             return new LogScope(this, message);
         }
 
-        internal void Log(string message)
+        public void Log(string message)
```

Comment:
You could also just make this a Shared source project, in which all files are effectively "statically linked" like we do in the Key Vault .NET SDK. No need to be public then.

Reply:
I'll do that!

Reply:
Thought about this -- I'll leave it as is for now.